RecoParticleFlow/PFTracking/interface/PFTrackTransformer.h

```diff
@@ -58,6 +58,8 @@ class PFTrackTransformer {
   ///B field
   math::XYZVector B_;
   PFGeometry pfGeometry_;
+  static constexpr double cos2ThetaV_Endcap_HiEnd_ = 0.99014;
```

Comment:
it would be nice to eventually drop this here and define the constants in BaseParticlePropagator, which is the origin of these magic numbers.

Reply:
@slava77 
Do you mean I define a constant in BaseParticlePropagator, and somehow use that in PFTrackTransformer as well?

Reply:
right. the value 0.99014 here is apparently supposed to match exactly the same value used (5 times) in BaseParticlePropagator.cc

Reply:
I will do this in the next PR. OK?

Reply:
OK.